spec/factories.rb

```diff
@@ -0,0 +1,18 @@
+require 'faker'
+FactoryGirl.define do
+  sequence :token do
+    SecureRandom.hex(3)
+  end
+
+  factory :event do
+    lat { Faker::Address.latitude.to_f.round(5) }
+    lon { Faker::Address.longitude.to_f.round(5) }
+    sequence(:name) { |n| "Name #{n}" }
+    sequence(:started_at) { |n| n.hours.from_now }
+    owner factory: :user
```

Comment:
So, there's another cool way to do this using FactoryGirl aliases.

`factory :user, alias [:owner] do`

and then can just rely on

```
     sequence(:started_at) { |n| n.hours.from_now }
     owner 
```

More: http://www.rubydoc.info/gems/factory_girl/file/GETTING_STARTED.md#Aliases